pkg/volume/gcepd/gce_util.go

```diff
@@ -81,7 +81,7 @@ func (util *GCEDiskUtil) DeleteVolume(d *gcePersistentDiskDeleter) error {
 		return err
 	}
 
-	if err = cloud.DeleteDisk(d.pdName); err != nil {
+	if err = cloud.DeleteDiskUnknownZone(d.pdName); err != nil {
```

Comment:
Need to rename DeleteDisk to DeleteDiskUnknownZone in pkg/volume/gcepd/attacher_test.go